<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setlist</class>
 <widget class="QDockWidget" name="Setlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>651</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Se&amp;tlist</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="4">
     <widget class="QListView" name="setListView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>A list of songs</string>
      </property>
      <property name="whatsThis">
       <string>A list of songs</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>364</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="addButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset theme="list-add">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="previousButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="nextButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QGroupBox" name="setupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Song Setup</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="preloadBox">
         <property name="toolTip">
          <string>Preload the song. If disabled, the song will only be loaded if it is active.</string>
         </property>
         <property name="text">
          <string>Preload</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameEdit">
         <property name="toolTip">
          <string>Name of the song</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Carla Patch</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KUrlRequester" name="patchrequester">
         <property name="toolTip">
          <string>Carla patch for the song</string>
         </property>
         <property name="filter">
          <string notr="true">*.carxp</string>
         </property>
         <property name="mode">
          <set>KFile::ExistingOnly|KFile::File|KFile::LocalOnly</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KUrlRequester" name="notesrequester">
         <property name="toolTip">
          <string>Notes or chords for the song</string>
         </property>
         <property name="filter">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
